nginx/default.conf

```diff
@@ -4,6 +4,28 @@ server {
     listen 8080;
     server_name _;
     client_max_body_size 1000M;
+
+    location /airflow/ {
+        proxy_set_header Host $host;
+        proxy_redirect off;
+        proxy_http_version 1.1;
+        proxy_set_header Upgrade $http_upgrade;
+        proxy_set_header Connection "upgrade";
+
+        # If the proxy_pass statement has no variables in it, then it will use the "gethostbyaddr" system call during start-up or reload and will cache that value permanently.
```

Comment:
Best to remove these comments from production code.

Reply:
removed